Assets/MRTK/Core/Definitions/SpatialAwareness/Experimental/SpatialAwarenessSceneObject.cs

```diff
@@ -62,26 +56,21 @@ public List<MeshData> Meshes
             private set;
         }
 
-        public struct MeshData
+        public class MeshData
         {
-            public System.Guid guid;
-            public int[] indices;
-            public Vector3[] vertices;
-            public Vector2[] uvs;
+            public int Id;
```

Comment:
Can these be properties instead of fields, to match our [encapsulation guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation)?